### ISSUE TITLE
Svelte-check fails to read paths from tsconfig

### ISSUE THREAD

**Issue by TeemuKoivisto** [sveltejs/language-tools#1544]
### Describe the bug

As I upgraded my `svelte-check` from 2.2.8 to 2.8.0 it doesn't resolve the paths from my tsconfig.json anymore:

```
{
  "compilerOptions": {
    ...
    "paths": {
      "$components": ["src/components/*"],
      "$context": ["src/context"],
      "$stores/*": ["src/stores/*"],
      "$tabs/*": ["src/tabs/*"],
      "$typings/*": ["src/typings/*"]
    },
  "include": ["src"],
  "exclude": ["**/__tests__"]
}
```

The errors are eg:

```
Error: Cannot find module '$components/Button.svelte' or its corresponding type declarations. (ts)
```

### Reproduction

I'm working on updating the packages here https://github.com/TeemuKoivisto/prosemirror-dev-toolkit/tree/master/core Just bumping the packages and running `yarn svelte-check` inside `core` should produce similar results. Running svelte-check without upgrading packages shouldn't produce those errors (but something else probably, haven't ran it in a while).

### Expected behaviour

It should work as before

### System Info

- OS: macOS
- IDE: VSCode

### Which package is the issue about?

svelte-check

**Comment by dummdidumm** [comment]
I'll look into this, but next time please at least put _SOME_ effort into your bug reports

> Running svelte-check without upgrading packages shouldn't produce those errors  (but something else probably, haven't ran it in a while)

Yeah maybe check that and don't let me check that.

> I upgraded my svelte-check from 2.2.8 to 2.8.0 it doesn't resolve the paths from my tsconfig.json anymore

How about checking at which version it broke?

**Comment by TeemuKoivisto** [comment]
I thought the problem was trivial so you wouldn't need much of premise. I put the version which introduced the change and would have done more if asked. Rollup bundles the package happily so you can imagine my confusion an arbitrary type-check package I added since it seems like a good practise starts to fail suddenly. As a maintainer myself I'm happy if someone even bothers to write a bug report and won't just first-hand complain about their shitty quality. It's always better to be courteous even if you think the other person is a moron.

**Comment by dummdidumm** [comment]
I agree that a low quality bug report is better than just shit talking the project. I think you were just lazy in this instance, that's all. We as maintainers all benefit from more complete bug reports because we can get to the cause of the bug faster.

**Comment by TeemuKoivisto** [comment]
Well again, a miscommunication. I meant as a maintainer myself I wont immediately shoot down a bad bug report but politely ask for better write-up. It's just being nice. And agree, I was lazy but I'm doing this on my own time so my focus is just getting things working to move on something else. Glad it was sorted out though! Will write a little more verbose in the future.

**Comment by dummdidumm** [comment]
👍🙂
